docs/known_issues.rst

```diff
@@ -4,17 +4,17 @@ Known Issues
 The following selection describes known bugs, errors, or other difficulties that
 may occur when using libEnsemble.
 
-* OpenMPI does not work with direct MPI job launches in mpi4py comms mode, since
-  it does not support nested MPI launches. Use either local mode or the Balsam job
-  controller.
+* OpenMPI does not work with direct MPI task launches in mpi4py comms mode, since
```

Comment:
Should this "launches" be "submissions"?

Reply:
Hm, I could see it as either "submissions" or "executions" depending if we want to emphasize the issues alongside the executor or just for OpenMPI. I might rewrite that issue a bit.